apis/task/tasks.py

```diff
@@ -0,0 +1,50 @@
+from flask import g, current_app
+from flask_restplus import Namespace, Resource
+from celery.result import AsyncResult
+from celery.exceptions import TimeoutError as Timeout
+import core.constants as C
+from .parsers import task_parser
+from .serializers import TaskResult
+
+api = Namespace('Tasks', description='Operations pertaining to celery asyncronous tasks')
+
+api.models['TaskResult'] = TaskResult
+
+@api.route('/<string:id>', methods=['GET'])
+class TaskRoute(Resource):
+    @api.expect(task_parser)
+    @api.marshal_with(TaskResult)
+    @api.response(200, C.CELERY_TASK_FETCHED)
+    @api.response(500, C.SERVER_ERROR)
```

Comment:
is there any response from the backend that indicates an invalid or unknown task id was supplied? Or do we care?